keras_hub/src/utils/transformers/export/hf_exporter.py

```diff
@@ -18,46 +21,39 @@
     # Add future models here, e.g., "LlamaBackbone": get_llama_weights_map,
 }
 
+MODEL_TOKENIZER_CONFIGS = {
+    "GemmaTokenizer": get_gemma_tokenizer_config,
+    # Add for future models, e.g., "LlamaTokenizer": get_llama_tokenizer_config,
+}
 
-def export_to_safetensors(keras_model, path):
-    """Converts a Keras model to Hugging Face safetensor format.
 
-    It does the following:
-    - Extracts and maps weights from the Keras backbone to safetensors.
-    - Saves the configuration as 'config.json'.
-    - Saves weights in 'model.safetensors'.
-    - Saves tokenizer assets.
+def export_backbone(backbone, path, include_lm_head=False):
+    """Export the backbone model to HuggingFace format.
 
     Args:
-        keras_model: The Keras model to convert.
-        path: str. Path of the directory to which the safetensors file,
-          config and tokenizer will be saved.
+        backbone: The Keras backbone model to convert.
+        path: str. Path to save the exported model.
+        include_lm_head: bool. If True, include lm_head weights if applicable.
     """
     backend = keras.config.backend()
-    backbone = keras_model.backbone
     model_type = backbone.__class__.__name__
-
     if model_type not in MODEL_CONFIGS:
         raise ValueError(f"Config not implemented for {model_type}")
```

Comment:
For all of these error messages (including the tokenizer one below), let's try to make these a little more self explanatory to the user.

`Export to transformer format not yet implemented for {model_type}.` Or something like that.